packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx

```diff
@@ -40,6 +41,8 @@ interface ISchemaAndDescriptionProps {
 
 const SchemaAndDescription = ({ title: titleProp, schema }: ISchemaAndDescriptionProps) => {
   const [resolveRef, maxRefDepth] = useSchemaInlineRefResolver();
+  const { renderExtensionAddon } = useOptionsCtx();
```

Comment:
Updated to allow schemas referred in the `Article`-component also work with the vendor extension renderers